subprojects/testfx-core/src/main/java/org/testfx/api/FxRobotInterface.java

```diff
@@ -187,8 +187,45 @@ public PointQuery offset(String query,
     // METHODS FOR INTERACTION AND INTERRUPTION.
     //---------------------------------------------------------------------------------------------
 
+    /**
+	 * Calls a runnable on the FX application thread and waits for it and consecutive events to
```

Comment:
The JavaDoc comment leading stars look misaligned, they should be aligned like so:

```java
/**
 * 
 * Blah blah.
 *
 * @param blah
 * @return blah
 */
```